src/Controller/CRUDController.php

```diff
@@ -866,38 +866,62 @@ public function historyCompareRevisionsAction($id = null, $base_revision = null,
 
         $reader = $manager->getReader($this->admin->getClass());
 
+        // NEXT_MAJOR: Remove this condition.
```

Comment:
Can you provide a test that verifies this hack?

Reply:
Done.